### ISSUE TITLE
Does not work after switching to Wayland

### ISSUE THREAD

**Issue by NicolasGoeddel** [mrvladus/Errands#308]
I just switched to Wayland and Errands does not show its UI anymore. If I start it from the terminal I can see that it loads and synchronizes tasks but the UI is just an icon without a window.

Here are few lines in the output that might help:
```
libEGL warning: egl: failed to create dri2 screen
MESA: error: CreateSwapchainKHR failed with VK_ERROR_INITIALIZATION_FAILED
MESA: error: zink: could not create swapchain
```
Any suggestions what might be the issue here?

**Comment by mrvladus** [comment]
I only use Wayland and never seen this error. What distro you are using? What GPU? And do you use Flatpak version?

**Comment by NicolasGoeddel** [comment]
Yes, I installed Errands as flatpak package. I am running Ubuntu 22.04.4 LTS with an Nvidia GeForce RTX 3080 TI and the proprietary drivers.
I just tried switching to Wayland on Friday by switching from "Gnome Xorg" to "Ubuntu Wayland" in the login screen. I already switched back shortly after because besides Errands not working I also had the issue with Firefox using a Software renderer for compositing. So there might be other issues here as well. And I don't know where to start to debug them.
